maven-surefire-common/src/test/java/org/apache/maven/plugin/surefire/AbstractSurefireMojoToolchainsTest.java

```diff
@@ -37,33 +37,91 @@
 
 import static java.util.Collections.emptyMap;
 import static java.util.Collections.singletonList;
+import static java.util.Collections.singletonMap;
+import static junit.framework.TestCase.assertNull;
 import static org.fest.assertions.Assertions.assertThat;
-import static org.fest.assertions.Fail.fail;
 import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
 import static org.powermock.reflect.Whitebox.invokeMethod;
 
 /**
- * Test for {@link AbstractSurefireMojo}.
+ * Test for {@link AbstractSurefireMojo}. jdkToolchain parameter
  */
 @RunWith( PowerMockRunner.class )
 @PrepareForTest( {AbstractSurefireMojo.class, ResolvePathsRequest.class, ReflectionUtils.class} )
```

Comment:
Here should be only
`@PrepareForTest( AbstractSurefireMojo.class )`
due to we are calling only one type, see `mockStatic( AbstractSurefireMojo.class )`.

Reply:
The performance of the test is mostly affected by this annotation.

Reply:
So, having only one class that is realy used is the right minimum and worth for the performance.

Reply:
Makes sense, thanks!